### ISSUE TITLE
Outgoing calls need a prefix number

### ISSUE THREAD

**Issue by donmanolito** [jitsi/jitsi#243]
Hello

I need to dail a 0 in order to call out. So right now I have all my contacts with an additional 0 on the contact list. The Problem is that the incomming calls are not refered to to contacts on the list and I can't just call back. 

Is it possible to add this 0 prefix to all calls if they have more that 3 digits (internal numbers have 3 digits)?

Best Regards

**Comment by ibauersachs** [comment]
No, this is not supported. Modern SIP PBXs are usually configured to handle internal and external calls transparently.

If someone wants to impelement this, a pull request is welcome, but it is very unlikely that an existing developer will take this.
